.eslintcache

```diff
@@ -0,0 +1 @@
+[{"C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\App.js":"1","C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\assets\\index.js":"2","C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\index.js":"3","C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\components\\profile-modal.jsx":"4","C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\mentors.js":"5"},{"size":4396,"mtime":1612705075676,"results":"6","hashOfConfig":"7"},{"size":166,"mtime":1612226601493,"results":"8","hashOfConfig":"7"},{"size":599,"mtime":1612226601507,"results":"9","hashOfConfig":"7"},{"size":2449,"mtime":1612226601502,"results":"10","hashOfConfig":"7"},{"size":593,"mtime":1612702560136,"results":"11","hashOfConfig":"7"},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1f3cm9g",{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"20","messages":"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\App.js",[],"C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\assets\\index.js",[],"C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\index.js",[],"C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\components\\profile-modal.jsx",[],"C:\\Users\\Raziq\\Programming\\Projects\\mentorship-page\\src\\mentors.js",[]]
```

Comment:
I think `.eslintcache` should be removed from this PR, since we aren't using ESLint. I suppose this was generated by your IDE? :P